src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -133,26 +134,26 @@ private static void setupDrivers(Application application) {
 		driverFeature.addDriver("Flip Vertical", flipVerticalTransformationDriver);
 
 		CompositeDriver compositeDriver = new CompositeDriver();
-		compositeDriver.add(loggerDriver);
-		compositeDriver.add(driver);
-		driverFeature.addDriver("Composite Driver", compositeDriver);
+		Job2dDriver usageDriver = new UsageDriver();
 
-		MouseClickAdapter mouseClickAdapter = new MouseClickAdapter(application.getFreePanel(), driverFeature.getDriverManager());
+		application.addComponentMenu(Job2dDriver.class, "Drivers utils");
+		application.addComponentMenuElementWithCheckBox(Job2dDriver.class, "LoggerDriver",  (ActionEvent e) -> compositeDriver.add(loggerDriver), false);
+		application.addComponentMenuElementWithCheckBox(Job2dDriver.class, "UsageDriver", (ActionEvent e) -> compositeDriver.add(usageDriver), false );
 
+		MouseClickAdapter mouseClickAdapter = new MouseClickAdapter(application.getFreePanel(), driverFeature.getDriverManager());
 		mouseClickAdapter.enable();
 	}
 	private static void setupMonitoringDeviceTests(Application application) {
+
 		DriverFeature driverFeature = FeatureManager.getFeature(DriverFeature.class);
 		DrawerFeature drawerFeature = FeatureManager.getFeature(DrawerFeature.class);
-
 		application.addTest("Monitoring device Test", new SelectMonitoringDeviceTestFigureOptionListener(driverFeature.getDriverManager()));
 		DrawPanelController drawerController = drawerFeature.getDrawerController();
-
+		/*
```

Comment:
If the code is not necessary right now. It should be removed. However, IMO the whole method is a complate functionality and it doesn't make sense to comment only its fragment. It would be simpler to remove setupMonitoringDeviceTests method call (which should disable it).